llvm/include/llvm/CAS/CASDB.h

```diff
@@ -391,6 +391,9 @@ void getDefaultOnDiskCASStableID(SmallVectorImpl<char> &Path);
 /// FIXME: Remove.
 std::string getDefaultOnDiskCASStableID();
 
+/// Create CASDB from a string identifier.
+Expected<std::unique_ptr<CASDB>> createCASFromIdentifier(StringRef Path);
```

Comment:
We should document the URI schemes this supports.

Reply:
This is just a quick hack to integrate any CAS in with current options. Any other suggestion is welcomed.